iree/turbine/kernel/wave/decompose_scan_ops.py

```diff
@@ -28,61 +32,134 @@ def get_graph_node(custom: CustomOp, graph: fx.Graph) -> fx.Node:
     return custom.fx_node
 
 
+def emit_local_inclusive_scan(
+    binary_fn: Callable,
+    scan_src: fx.Node,
+    graph: fx.Graph,
+    elements_per_thread: int,
+) -> list[fx.Node]:
+    """
+    Perform local inclusive scan for `n` elements per thread.
+    """
+    values = [
+        get_graph_node(Extract(scan_src, [i]), graph)
+        for i in range(elements_per_thread)
+    ]
+
+    for i in range(1, elements_per_thread):
+        values[i] = get_graph_node(binary_fn(values[i], values[i - 1]), graph)
+
+    return values
+
+
 def emit_global_scan(
     binary_fn: Callable,  # Supports only Add for now.
     src: fx.Node,
+    local_scan: list[fx.Node],
```

Comment:
Could you pass in just the fx.Node instead of accessing the last element of the list when the rest of the elements in the list are no longer required?